api/src/pcapi/core/fraud/factories.py

```diff
@@ -86,7 +85,7 @@ class Meta:
     postal_code = "75008"
     procedure_number = factory.Faker("pyint")
     registration_datetime = LazyAttribute(
-        lambda _: datetime.utcnow().replace(tzinfo=pytz.utc).strftime("%Y-%m-%dT%H:%M:%S%z")
+        lambda _: datetime.utcnow().replace(tzinfo=None).strftime("%Y-%m-%dT%H:%M:%S%z")
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏
```suggestion
        lambda _: datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%S%z")
``` 
Fait la même chose